server/documents/modules/modal.html.eco

```diff
@@ -509,6 +526,25 @@ themes      : ['Default', 'Material']
       </div>
     </div>
 
+    <div class="no example">
+      <h4 class="ui header">Inverted Variant</h4>
+      <p>A modal can be shown inverted on white dimmer...</p>
+      <div class="code" data-demo="true" data-eval="$('.mini.test.inverted.modal').not('.basic').modal({inverted:true}).modal('show')">
+      $('.mini.inverted.modal')
+        .modal({
+           inverted: true
+        })
+        .modal('show')
+      ;
+      </div>
+        <p>... or default black dimmer</p>
+      <div class="code" data-demo="true" data-eval="$('.mini.test.inverted.modal').not('.basic').modal('show')">
```

Comment:
After opening the the "white dimmer" variation, running this always shows the white dimmer.
![peek 2018-10-16 07-56](https://user-images.githubusercontent.com/5517677/46995437-16b2bc00-d119-11e8-92e9-31dfee2844a8.gif)